openapi/openapi-client/src/main/openapi3/master_rest_v1.yaml

```diff
@@ -253,16 +253,20 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ApplicationsHeartbeatResponse'
+
+  /api/v1/applications/{appIds}:
```

Comment:
IMO, we can use `POST` method instead of `DELETE` for batch delete operations.

Reply:
done